drf_spectacular/templates/drf_spectacular/swagger_ui.html

```diff
@@ -10,7 +10,9 @@
     <div id="swagger-ui"></div>
     <script src="//unpkg.com/swagger-ui-dist@3.35.0/swagger-ui-bundle.js"></script>
     <script>
+    var swagger_settings  = {{ settings|safe }}
     const ui = SwaggerUIBundle({
+        ...swagger_settings,
```

Comment:
i think it makes more sense to have `...swagger_settings` at the bottom so that people have the ability override existing settings.